lib/shared-method.js

```diff
@@ -369,7 +369,9 @@ function coerceAccepts(uarg, desc) {
   if (actualType === 'number' && Number.isNaN(uarg)) {
     throw new badArgumentError(name + ' must be a number');
   }
-
+  if (actualType === 'integer' && Number.isNaN(uarg)) {
```

Comment:
No, as there is already a check for the data type at [line#352](https://github.com/strongloop/strong-remoting/pull/323/files#diff-157ee1a11c353da754b459ce1fe0bc0dR352). This is for `NaN` values only.